llvm/lib/Transforms/InstCombine/InstCombineInternal.h

```diff
@@ -238,6 +238,8 @@ class LLVM_LIBRARY_VISIBILITY InstCombinerImpl final
   std::optional<std::pair<Intrinsic::ID, SmallVector<Value *, 3>>>
   convertOrOfShiftsToFunnelShift(Instruction &Or);
 
+  Value *EmitGEPOffset(GEPOperator *GEP, bool RewriteGEP = false);
```

Comment:
Why did this move?

Reply:
Oh - This was a relic of a previous version that moved the creation of geps into (a differently named) shouldCanonicalizeGEPToPtrAdd, but it was simpler to just have the condition in that function.